tools/bin/gpcheck

```diff
@@ -567,7 +683,7 @@ def testHAWQ(host):
             return
 
         # check HAWQ master's memory size
-        expected_vmemory_size = 1024
+        expected_vmemory_size = 8192
```

Comment:
do we want to check exact value, or minimal value ? in other words, should it be guc_vmemsize_master < expected_vmemory_size rather than guc_vmemsize_master != expected_vmemory_size in the statement below and other similar comparisons ?